features/insight_summary_page.feature

```diff
@@ -20,3 +20,43 @@ Feature: Evidence Hub: Insight Summary page
       | qualitative  | tick    |
       | quantitative | cross   |
+
+  Scenario: Key Info client_group filters
+    Given I entered into the "Insight" page "Financial well-being: the employee view"
+    When I click on the "Over-indebted people" client group in the key info panel
+    Then I should return to the search results page
+    And I should see "2" evidence summaries
+    And I should see the "first" evidence summary as
+      | Field               | Value                                   |
+      | document title      | Financial well-being: the employee view |
+      | evidence type       | Insight                                 |
+      | topics              | Topics: Saving, Financial Capability    |
+      | countries           | United Kingdom                          |
+      | year of publication | 2015                                    |
+    And I should see the "second" evidence summary as
+      | Field               | Value                                                              |
+      | document title      | Moving forward together: peer support for people with problem debt |
+      | evidence type       | Insight                                                            |
+      | topics              | Credit Use and Debt                                                |
+      | countries           | England                                                            |
+      | year of publication | 2017                                                               |
+  
+  Scenario: Key Info topic filters
+    Given I entered into the "Insight" page "Financial well-being: the employee view"
+    When I click on the "Saving" topic in the key info panel
+    Then I should return to the search results page
+    And I should see "2" evidence summaries
+    And I should see the "first" evidence summary as
+      | Field               | Value                                   |
+      | document title      | Financial well-being: the employee view |
+      | evidence type       | Insight                                 |
+      | topics              | Topics: Saving, Financial Capability    |
+      | countries           | United Kingdom                          |
+      | year of publication | 2015                                    |
+    And I should see the "second" evidence summary as
+      | Field               | Value                                                                 |
+      | document title      | Raising household saving                                              |
+      | evidence type       | Review                                                                |
+      | topics              | Topics: Saving, Pensions and Retirement Planning, Financial Education |
+      | countries           | International review                                                  |
+      | year of publication | 2012                                                                  |
```

Comment:
You are changing the view evidence_summaries/show, then the feature is applied to evaluation and review. I think is worth to add tests for Evaluation and review. If this is the case, can we summarise this steps into one? Then will be less repetitive for test evaluation and review pages as well. Wdyt?

Reply:
Originally I intended to do the 3 different types of evidence summary as separate prs since they are separate cards, but after further thought I have decided to do all 3 in one pr since there is alot of overlap.